tikv-configuration-file.md

```diff
@@ -398,6 +398,23 @@ Configuration items related to the sharing of block cache among multiple RocksDB
 + Default value: 45% of the size of total system memory
 + Unit: KB|MB|GB
 
+### `api-version` <span class="version-mark">New in v6.1.0</span>
+
++ The storage format and interface version used by TiKV when TiKV serves as the raw key-value store.
++ Value options:
+    + `1`: Uses API V1, does not encode the data passed to the client, and stores data as it is. In versions earlier than v6.1.0, TiKV uses API V1 by default.
+    + `2`: Uses API V2:
+        + The data is stored in the MVCC (Multi Version Concurrency Control) format, where the timestamp (TSO) is obtained from PD by tikv-server.
+        + When API V2 is used, you are expected to set `storage.enable-ttl = true` at the same time. Because API V2 supports the TTL feature, it is mandatory to turn on `enable-ttl` in case that `enable-ttl` appears in contradiction with API V2.
+        + When API V2 is enabled, you need to deploy at least one tidb-server instance to reclaim expired data. Note that this tidb-server instance cannot provide read or write services. To ensure high availability, you can deploy multiple tidb-server instances.
+        + Client support is required for API V2. For details, see the corresponding instruction of the client for the API V2.
++ Default value: `1`
+
+> **Warning:**
+>
+> - You can set the value of `api-version` to `2` **only when** deploying a new TiKV cluster. **Do not** modify the value of this configuration item in an existing TiKV cluster. TiKV clusters with different `api-version` values use different data formats. Therefore, if you modify the value of this item in an existing TiKV cluster, the cluster will store data in different formats, which causes data corruption and the "unable to switch storage.api_version" error when you start the TiKV cluster.
```

Comment:
```suggestion
> - You can set the value of `api-version` to `2` **only when** deploying a new TiKV cluster. **Do not** modify the value of this configuration item in an existing TiKV cluster. TiKV clusters with different `api-version` values use different data formats. Therefore, if you modify the value of this item in an existing TiKV cluster, the cluster will store data in different formats and causes data corruption. It will raise the "unable to switch storage.api_version" error when you start the TiKV cluster.
```

It will raise the error on startup to prevent data corruption actually.